src/actions/search.js

```diff
@@ -0,0 +1,18 @@
+import axios from '../utils/axiosInstance';
+import { SEARCH } from './types';
+
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Why are you disabling eslint rules here?

Reply:
I thought I would be adding more action creators in this file and forgot to remove it.

Reply:
It's resolved.